modules/authloginpass/locales/en_US/auth.UTF-8.properties

```diff
@@ -4,3 +4,67 @@ message.failedToLogin = Failed to login. The login or the password is invalid.
 form.login                 = login
 form.password              = password
 form.buttons.login         = login
+form.password.forget       = Forgot my password
+
+password.reset.subject=Reset of your password on the web site %s
+password.reset.body.html=<p>Hello,</p>\n\n\
```

Comment:
I would prefer you keep the original mail.UTF-8.properties and password.UTF-8.properties, instead of merging them into this auth.UTF-8.properties. This is important for mail.UTF-8.properties, because it is easier to redefine mail content (the developer do not have to copy all other locales if he just want to change emails content).